mobile/lib/modules/home/ui/profile_drawer/profile_drawer_header.dart

```diff
@@ -19,7 +18,7 @@ class ProfileDrawerHeader extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    String endpoint = Hive.box(userInfoBox).get(serverEndpointKey);
+    final String? endpoint = Store.get(StoreKey.serverEndpoint);
```

Comment:
`final String endpoint` since `Store.get` will throw an exception if the value is null. Maybe you wanted `tryGet`.

Reply:
removed the `?`